src/features/people/PersonPage/Movies/index.js

```diff
@@ -13,7 +13,7 @@ export const Movies = (props) => {
     switch (fetchStatus) {
         case "completed":
             return (
-                <Content>
+                <>
                     <Title>Movies - {header} ({list.length})</Title>
```

Comment:
I think Title component could be replaced with Header component we created in common folder.

Reply:
fixed